apps/api/src/app.module.ts

```diff
@@ -18,9 +18,7 @@ import { RatingModule } from './rating/rating.module';
   imports: [
     //use for local db test: mongodb://localhost:27017/test
     //use for central db test: mongodb+srv://user:aZkh7DhYjrP2i52U@personalproject.fzwyrwp.mongodb.net/POC-EYCommerce
-    MongooseModule.forRoot(
-      'mongodb+srv://user:aZkh7DhYjrP2i52U@personalproject.fzwyrwp.mongodb.net/POC-EYCommerce',
-    ),
+    MongooseModule.forRoot('mongodb://localhost:27017/test'),
```

Comment:
make sure to use the connection string for central db before merging.

Reply:
this is noted thanks